static/css/components/preview.less

```diff
@@ -4,8 +4,8 @@
 
 .book-preview {
   .iframe-container {
-    border-top: 1px solid #eee;
-    border-bottom: 1px solid #eee;
+    border-top: 1px solid rgb(238, 238, 238);
```

Comment:
#eee is equivalent to hsl(0, 0%, 93.3%); which is the variable #4550 `@lightest-grey` so please use the variable here.